Platform/OpenUsbKbDxe/KeyBoard.c

```diff
@@ -520,9 +520,9 @@ InitKeyboardLayout (
   EFI_KEY_DESCRIPTOR        *TableEntry;
   EFI_KEY_DESCRIPTOR        *NsKey;
   USB_NS_KEY                *UsbNsKey;
-  UINTN                     Index;
-  UINTN                     Index2;
-  UINTN                     KeyCount;
+  UINT8                     Index;
+  UINT8                     Index2;
+  UINT8                     KeyCount;
```

Comment:
Is this change reducing type sizes because USB_KEYBOARD_LAYOUT_PACK_BIN DescriptorCount is UINT8? Can be a good idea, but I believe at least indices would want some suffix to denote that it is not a normal Index of UINTN type. It has happened for me when I reused an Index variable of a different type in a loop assuming it was UINTN by mistake.

Reply:
EDK II codestyle requires capitalised local variables, so better use something like `DescIndex` and `DescIndex2` probably.